.github/workflows/copilot-setup-steps.yml

```diff
@@ -17,5 +17,66 @@ jobs:
     steps:
       - uses: actions/checkout@v4.2.2
 
+      # From https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/main/src/ubuntu/25.04/helix/Dockerfile
+      - name: Install dependencies
+        env:
+          LIBMSQUIC_VERSION: '2.4.8'
+        run: |
+          sudo apt-get update -y && \
+          sudo apt-get install -qq -y \
+          autoconf \
+          automake \
```

Comment:
Consider `eng/common/native/install-dependencies.sh linux` to avoid duplicating this